core/src/test/java/org/pentaho/platform/web/http/api/resources/SchedulerResourceTest.java

```diff
@@ -144,12 +144,8 @@ public void testCreateJobError() throws Exception {
 
     testResponse = schedulerResource.createJob( mockRequest );
     assertEquals( mockForbiddenResponse, testResponse );
-
-    verify( mockSchedulerException, times( 1 ) ).getCause();
-    verify( mockSchedulerExceptionCause, times( 1 ) ).getMessage();
```

Comment:
checking for number times an exception.getCausse().getMessage() is to fine grain/brittle already very buildServerErrorResponse(string) gets called and verifies the correct exception argument string gets called.